src/panels/calendar/ha-full-calendar.ts

```diff
@@ -124,26 +127,27 @@ export class HAFullCalendar extends LitElement {
                           "ui.components.calendar.today"
                         )}</mwc-button
                       >
-                      <ha-icon-button
+                      <ha-icon-button-prev
                         .label=${this.hass.localize("ui.common.previous")}
                         .path=${mdiChevronLeft}
                         class="prev"
                         @click=${this._handlePrev}
                       >
-                      </ha-icon-button>
-                      <ha-icon-button
+                      </ha-icon-button-prev>
+                      <ha-icon-button-next
                         .label=${this.hass.localize("ui.common.next")}
                         .path=${mdiChevronRight}
```

Comment:
Not needed now

Reply:
@zsarnett Why is it not needed? I created the -prev/-next components a few years back so they automatically adapt to RTL...

Reply:
The path isnt needed right?

Reply:
Oh right! Removed. The comment was on different lines so I didn't follow. Thanks.